test/manual/tls_support.test.ts

```diff
@@ -77,19 +77,17 @@ describe('TLS Support', function () {
 
         it('sets the default options', async function () {
           await client.connect();
-          expect(tlsSpy).to.have.been.calledWith({
-            autoSelectFamily: true,
-            host: 'localhost',
-            port: 27017,
-            servername: 'localhost',
-            ca: sinon.match.defined,
-            cert: sinon.match.defined,
-            key: sinon.match.defined
-          });
+          expect(tlsSpy).to.have.been.calledWith(
```

Comment:
Same here narrowing assertions so we don't break unrelated tests when changing connection options.